### ISSUE TITLE
Still questions about size = 2048

### ISSUE THREAD

**Issue by Senwang98** [olly-styles/Multiple-Object-Forecasting#12]
Hi, olly
I suspect that you have modified the DTP, because the original DTP output is a correction value, but the current 2048 does not mean this at all. For this part, can you give more details? If you  can,  git push code about generating dtp_feature will be much better. @olly-styles
Thanks.

**Comment by Senwang98** [comment]
Just train the STED with ResNet18 together OR train ResNet18 first then train the left encoder and decoder???
Because the output of ResNet is 2048, I want to ask if you have trained ResNet and other modules of STED as a whole, or if you trained ResNet separately, then go to train other modules of STED.
This is important for me, thanks.

**Comment by olly-styles** [comment]
"I suspect that you have modified the DTP, because the original DTP output is a correction value, but the current 2048 does not mean this at all" 

We use the feature vector from the final penultimate layer of DTP, not the output layer.

"train the STED with ResNet18 together OR train ResNet18 first then train the left encoder and decoder?"

We train the optical flow encoder first and freeze these weights, as described in Section 6.2 of the paper.  - "We train the optical flow feature encoder using ResNet50 [14] as the backbone CNN architecture for 10k iterations with a batch size of 64 and learning rate of 1 × 10−5 to predict future object locations as described in [40] and then freeze the weights to use our flow encoder as a fixed feature extractor."

**Comment by Senwang98** [comment]
Ok, I see, This question confused me for a week, thanks olly
